process/smartContract/hooks/argHooks.go

```diff
@@ -0,0 +1,21 @@
+package hooks
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/data/typeConverters"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+)
+
+// ArgBlockChainHook represents the arguments structure for the blockchain hook
+type ArgBlockChainHook struct {
```

Comment:
Call it maybe BlockChainHookArg

Reply:
I followed the arguments naming convention already in use.

Reply:
ok... should be called with Arg at the end in the future